spring-boot-sample-project/src/main/java/com/example/demo/service/UserService.java

```diff
@@ -0,0 +1,58 @@
+package com.example.demo.service;
+
+import com.example.demo.pojo.User;
+import com.example.demo.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+public class UserService {
+
+
+    private UserRepository userRepository = new UserRepository();
```

Comment:
You should use `@AutoWired` and not instantiate yourself. This is the point of DI